.travis.yml

```diff
@@ -10,7 +10,7 @@ env:
   - DISPLAY=:99
 script:
   - mvn clean test -Dgpg.skip=true -Dgalen.use.fail.exit.code=false
-  - mvn test-compile failsafe:integration-test failsafe:verify -Dgpg.skip=true -Dgalen.use.fail.exit.code=false
+  - mvn verify -Dgpg.skip=true -Dgalen.use.fail.exit.code=false
```

Comment:
I think that it didn't work because the verify goal was not part of the failsafe goals in the pom.xml.